C#/syntax_test_c#.cs

```diff
@@ -73,3 +73,41 @@ int Zoo()
     //  ^ entity.name.function
     {}
 }
+
+class Syntax
+{
+    public decimal decimal1 = 1.0m;
+    //                        ^^^^ constant.numeric.source.cs
+    public decimal decimal2 = 2.0M;
+    //                        ^^^^ constant.numeric.source.cs
+    public double double1 = 1.0d;
+    //                      ^^^^ constant.numeric.source.cs
+    public double double2 = 2.0D;
+    //                      ^^^^ constant.numeric.source.cs
+    public float float1 = 1.0f;
+    //                    ^^^^ constant.numeric.source.cs
+    public float float2 = 2.0F;
+    //                    ^^^^ constant.numeric.source.cs
+    public long long1 = 1l;
+    //                  ^^ constant.numeric.source.cs
+    public long long2 = 2l;
+    //                  ^^ constant.numeric.source.cs
+    public ulong ulong1 = 1ul;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong ulong2 = 2UL;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong ulong3 = 3lu;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong ulong4 = 4LU;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong ulong5 = 5uL;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong ulong6 = 6Ul;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong ulong7 = 7lU;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong ulong8 = 8Lu;
+    //                    ^^^ constant.numeric.source.cs
+    public ulong bad = 1UU;
+    //                 ^^^ meta.class.body.source.cs
```

Comment:
This should probably be changed to `- constant.numeric` if you want to check and make sure it is not recognized as a suffix.